packages/storage-plus/src/keys.rs

```diff
@@ -38,8 +40,9 @@ impl<'a> PrimaryKey<'a> for &'a [u8] {
 // Provide a string version of this to raw encode strings
 impl<'a> PrimaryKey<'a> for &'a str {
     type Prefix = ();
+    type SubPrefix = ();
```

Comment:
Simple. Just add the 2-step variant rather than some generic N-step construction. Should be plenty for any case I can think of.